pennylane/ops/op_math/sum.py

```diff
@@ -291,3 +294,33 @@ def queue(self, context=qml.QueuingContext):
         for op in self.summands:
             context.safe_update_info(op, owner=self)
         return self
+
+    @property
+    def arithmetic_depth(self) -> int:
+        return 1 + max(summand.arithmetic_depth for summand in self.summands)
+
+    def simplify_summands(self, depth=-1) -> List[Operator]:
```

Comment:
We might reserve `simplify` for the same action that the current `simplify` method does.

Reply:
I'm fine with the name `simplify_summands`.

But I do think we should make it a private method `_simplify_summands`.  That gives us more flexibility to change it in the future, and it should only ever be accessed from `op.simplify` anyway.

Reply:
Changed the method name to `_simplify_summands`. I also changed it to a `@classmethod` to avoid a `protected-access` pylint error. (see commit b1787b6eb7fb8b1a58bc7db971571c442de96a20)